client/me/purchases/manage-purchase/index.jsx

```diff
@@ -67,14 +67,10 @@ function isDataLoading( props ) {
 
 const ManagePurchase = React.createClass( {
 	propTypes: {
-		cart: React.PropTypes.object.isRequired,
-		selectedPurchase: React.PropTypes.object.isRequired,
-		selectedSite: React.PropTypes.oneOfType( [
-			React.PropTypes.object,
-			React.PropTypes.bool,
-			React.PropTypes.undefined
-		] ),
 		destinationType: React.PropTypes.string,
+		hasLoadedSites: React.PropTypes.bool.isRequired,
+		selectedPurchase: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.object.isRequired,
```

Comment:
I believe the previous three types for `selectedSite` was accurate, for when the selected site is loading, present, or when `sites-list` has been fetched but the site in the URL isn't present.